src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -3685,6 +3685,9 @@
   <data name="InvalidOperation_EditFailed" xml:space="preserve">
     <value>The assembly update failed.</value>
   </data>
+  <data name="NotSupported_DebuggerAttached" xml:space="preserve">
+    <value>Assembly updates not supported while debuggger is attached.</value>
```

Comment:
Maybe something like:
```suggestion
    <value>Updates were applied to an assembly in the process. The debugger is unable to attach.</value>
```
?

Reply:
Oh, wait, nevermind, this error message is about the debugger already being attached?